inst/templates/ad_adrs_basic.R

```diff
@@ -225,7 +229,7 @@ adrs <- adrs %>%
       PARAM = "Confirmed Response by Investigator",
```

Comment:
I received an error, even though I am using updated version of derive_param_confirmed_resp() function

Error in filter_joined(dataset, dataset_add = dataset, by_vars = subject_keys,  : 
  unused arguments (dataset_add = dataset, filter_join = AVALC == "CR" & AVALC.join == "PR")